### PR TITLE
DIV-4852 Add jquery hook for on click submit button to disable it

### PR DESCRIPTION
**Pull request by alexwakeman**
# Description

Ensure that every submit button is disabled after clicking it.

Fixes # DIV-4852

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Manual testing

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -5,4 +5,12 @@ import './documentUpload';
...
   const showHideContent = new ShowHideContent();
   showHideContent.init();
+
+  $('input.button[type="submit"]').click(event => {
+    const $el = $(event.target);
+    setTimeout(() => {
+      $el.attr('disabled', true);
+      $el.attr('aria-disabled', true);
+    });
+  });
```
Comment (hosainnet) [inline assets/js/main.js]: It'd be good to have a test for this

Comment (alexwakeman) [reply assets/js/main.js]: It has been tested - manually 😄 
To really test it we would need Codecept to enable throttling of network connection, as the disable happens after submissions. 
If you see here: https://github.com/hmcts/div-respondent-frontend/blob/38c9d2004d37282bcb41b15823a4c1652913052b/test/e2e/helpers/JSWait.js#L21
We are forcing the e2e tests to wait for the next page to load which means it's too late to test if the button is disabled. 
We're not testing it in PFE currently.
If you can suggest a way to test I can look into it. Unit testing it seems unnecessary as it is all based on library code (jQuery) and not our own code.

Comment (alexwakeman) [reply assets/js/main.js]: I could add a small customised e2e test to check it actually. Let me have a look now.

Comment (hosainnet) [reply assets/js/main.js]: Okay I can see how that might be flaky, I guess given that it's not that critical to the flow it might be okay as it is

Comment (alexwakeman) [reply assets/js/main.js]: Ok cool thanks @hosainnet

**Comment by nsingamsetty** [comment]
Tested and working as Expected.Refer ticket for evidences.Merging now.

**Pull request merged**